base/sparse/sparsevector.jl

```diff
@@ -636,8 +636,8 @@ function showarray(io::IO, x::AbstractSparseVector;
 end
 
 function summary(x::AbstractSparseVector)
-    string("Sparse vector of length ", length(x), ", with ", length(nonzeros(x)),
-           " ",  eltype(x), " nonzero entries:")
+    string(length(x), "-element sparse vector with ", length(nonzeros(x)),
```

Comment:
Yeah, maybe. OTOH, the fact that the number of `"entries"` is printed on the same line removes any ambiguity.

I could also reintroduce the term "nonzero entries", which isn't currently used for matrices, but which could be added for both matrices and vectors.